### PR TITLE
Wire up record types in sidebar

### PR DESCRIPTION
**Pull request by kshepard**
Will add tests in forthcoming PR.

### CODE REVIEW

File: schema_editor/app/scripts/views/recordtype/detail-add-edit-partial.html [hunk 1]
```diff
@@ -47,8 +47,8 @@ <h2 ng-show="rtDetail.schemaKey">Edit Related Content</h2>
                 <button type="submit" class="btn btn-default" ng-disabled="rtForm.$invalid">
                     Save
                 </button>
-                <button type="button" ui-sref="rtDetail.detail({uuid: rtDetail.recordType.uuid})" class="btn btn-default">Cancel</button>
+                <button type="button" ui-sref="rt.detail({uuid: rtDetail.recordType.uuid})" class="btn btn-default">Cancel</button>
```
Comment (CloudNiner) [inline schema_editor/app/scripts/views/recordtype/detail-add-edit-partial.html]: Double check to make sure this shouldn't be the other way around.

File: schema_editor/app/scripts/views/sidebar/sidebar-partial.html [hunk 2]
```diff
@@ -8,18 +8,11 @@
...
-            </li>
-            <li class="active-parent in">
-                <a>&#8212; Accidents</a>
+            <li class="active-parent in" ng-repeat="recordType in sb.recordTypes | orderBy:'label'" class="row">
+                <a>&#8212; {{ recordType.label }}</a>
                 <ul>
-                    <li><a ui-sref="rt.detail({uuid: 'abcd'})">View related content</a></li>
-                    <li><a ui-sref="rt.schema-add({uuid: 'abcd'})">Add related content</a></li>
+                    <li><a ui-sref="rt.detail({uuid: '{{recordType.uuid}}'})">View related content</a></li>
+                    <li><a ui-sref="rt.schema-add({uuid: '{{recordType.uuid}}'})">Add related content</a></li>
```
Comment (CloudNiner) [inline schema_editor/app/scripts/views/sidebar/sidebar-partial.html]: Oh, this should be `rt.detail-add(...)`

File: schema_editor/app/scripts/views/sidebar/sidebar-partial.html [hunk 3]
```diff
@@ -8,18 +8,11 @@
...
-            <li class="active-parent in">
-                <a>&#8212; Crimes</a>
-                <ul>
-                    <li><a ui-sref="rt.detail({uuid: 'abcd'})">View related content</a></li>
-                    <li><a ui-sref="rt.schema-add({uuid: 'abcd'})">Add related content</a></li>
-                </ul>
-            </li>
-            <li class="active-parent in">
-                <a>&#8212; Accidents</a>
+            <li class="active-parent in" ng-repeat="recordType in sb.recordTypes | orderBy:'label'" class="row">
```
Comment (CloudNiner) [inline schema_editor/app/scripts/views/sidebar/sidebar-partial.html]: There's also a small bit of functionality where clicking the header with the label collapses/expands the list. 

All that needs to be done for that to work is to toggle one of classes:`active-parent in` on click and leave that class off by default. I just don't remember which one.

**Comment by CloudNiner** [comment]
Check that travis build, our one `true === true` test is failing, looks like a dependency issue.

**Comment by CloudNiner** [comment]
+1 on minor cleanup/travis

File: schema_editor/app/scripts/views/recordtype/detail-add-edit-partial.html (hunk 1 above)
Comment (kshepard) [inline schema_editor/app/scripts/views/recordtype/detail-add-edit-partial.html]: Pretty sure this is correct. Seems to do what I'd expect.

File: schema_editor/app/scripts/views/sidebar/sidebar-partial.html (hunk 3 above)
Comment (kshepard) [inline schema_editor/app/scripts/views/sidebar/sidebar-partial.html]: Ah, I didn't notice that in the prototype. I'll add it.

**Pull request merged**